MSAL/src/native_auth/input_validator/MSALNativeAuthInputValidator.swift

```diff
@@ -26,10 +26,15 @@ import Foundation
 
 protocol MSALNativeAuthInputValidating {
     func isInputValid(_ input: String) -> Bool
+    func isInputValid(_ input: Array<Any>) -> Bool
```

Comment:
nit: swiftlint is complaining about this Syntactic sugar violation.
```suggestion
    func isInputValid(_ input: [Any]) -> Bool
```

Reply:
Done.